src/c++/perf_analyzer/mock_model_parser.h

```diff
@@ -39,6 +39,7 @@ class MockModelParser : public ModelParser {
       scheduler_type_ = ModelParser::SEQUENCE;
     }
     is_decoupled_ = is_decoupled_model;
+    max_batch_size_ = 64;
```

Comment:
Why add this?

Reply:
Without this, the default of max_batch_size is 0, which says there is no batching. That made it impossible for me to do batching testing.

For now, I'm going to change it to be an optional arg in the mock constructor. Later we can re-work it all to use gmock or something else as needed.